src/api/venue.ts

```diff
@@ -77,3 +79,65 @@ export const updateIframeUrl = async (iframeUrl: string, venueId?: string) => {
     .functions()
     .httpsCallable("venue-adminUpdateIframeUrl")({ venueId, iframeUrl });
 };
+
+const getUserInSectionRef = (
+  userId: string,
+  venueId: string,
+  sectionId: string
```

Comment:
use an object here

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2318/commits/f0c98b720e24314deebbf7593d97ad53147ba0d3